### ISSUE TITLE
New videos spinning loading wheel, won't play

### ISSUE THREAD

**Issue by mercertom** [ajayyy/SponsorBlock#1430]
Took me forever to isolate SponsorBlock, but disabling skipping in SB makes the video play. AFAICT, this is for new videos that have no SB information or segments yet, and something in SB's loading process makes the video never ready.

Opening in private tab makes the video play.

Often waiting around 1 day makes the video play (probably because people find segments in the video, or SB identifies the video somehow)

Disabling skipping makes the video play.

Logging out of YT account makes the video play.

So the video is obviously good to go from my browser's perspective.

Basically, something is wrong for videos before they get their 1st to-be-skipped segment identified by SB, and it prevents video playback until the 1st to-be-skipped segment is identified (or the other workarounds, which have obvious drawbacks).
